### PR TITLE
Deduction page

### PR DESCRIPTION
**Pull request by kaedm**
deduction page with create deduction modal

### CODE REVIEW

**Comment by kaedm** [comment]
1. Deduction page
image

2. Create deduction modal
image

File: app.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "payroll-system",
+  "name": "payroll-system_1",
   "type": "app",
-  "defaultPage": "payroll",
+  "defaultPage": "benefits",
```
Comment (torodeveloper) [inline app.json]: I thought payroll would be our default page. Why did you change it to benefits?

File: components/navigation/index.json
Comment (torodeveloper) [inline components/navigation/index.json:1]: The navigation-bar component is deleted again and this component is added again.

File: components/run-payroll/index.json
Comment (torodeveloper) [inline components/run-payroll/index.json:1]: You can't remove this run-payroll modal.

File: components/run-payslips/index.json
Comment (torodeveloper) [inline components/run-payslips/index.json:1]: You can't remove this run-payslips modal.

File: pages/payslips.json
Comment (torodeveloper) [inline pages/payslips.json:1]: You can't remove this payslips page.

**Review by torodeveloper — CHANGES_REQUESTED** [review]

File: app.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "payroll-system",
+  "name": "payroll-system_1",
```
Comment (torodeveloper) [inline app.json]:
```suggestion
  "name": "payroll-system",
```

File: components/new-role/stylesheets/new-role.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+new-role {
+
+}
+
+.small-input {
```
Comment (torodeveloper) [inline components/new-role/stylesheets/new-role.css]: Shouldn't it be?
```suggestion
new-role .small-input {
```

File: components/new-role/stylesheets/new-role.css [hunk 4]
```diff
@@ -0,0 +1,7 @@
+new-role {
```
Comment (torodeveloper) [inline components/new-role/stylesheets/new-role.css]: Please remove empty css declaration.

Comment (kaedm) [reply components/new-role/stylesheets/new-role.css]: image

**Review by torodeveloper — APPROVED** [review]

**Pull request merged**